jisantuc/AFDG
# Add wall toggles

Overview
-----

This PR makes it so that walls can be added to and removed from tiles if
it's the appropriate mode. It also fixes a bug in the reachability logic that
I discovered in testing adding and removing walls.

Testing
-----

- do "add walls" or "remove walls" mode
- click on some dotted lines to add walls and some present walls to remove them
- verify that they successfully are added and removed
- check reachability based on your changes and confirm it makes sense

Closes #18
